gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/sampling/statistics/AverageClusteringCoefficientTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.statistics;
+
+import org.apache.commons.math3.util.Precision;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test-class for {@link AverageClusteringCoefficient}
+ */
+public class AverageClusteringCoefficientTest extends GradoopFlinkTestBase {
+
+  /**
+   * GraphLoader used for test-cases
+   */
+  protected FlinkAsciiGraphLoader loader;
+
+  /**
+   * Constructor
+   */
+  public AverageClusteringCoefficientTest() {
+
+    String graphString =
+      "/* vertices */" +
+      "(v0 {id:0, value:\"A\"})" +
+      "(v1 {id:1, value:\"B\"})" +
+      "(v2 {id:2, value:\"C\"})" +
+      "clique[" +
+      "/* fully connected clique */" +
+      "(v0)-[e0]->(v1)" +
+      "(v1)-[e1]->(v0)" +
+      "(v0)-[e2]->(v2)" +
+      "(v2)-[e3]->(v0)" +
+      "(v1)-[e4]->(v2)" +
+      "(v2)-[e5]->(v1)" +
+      "]" +
+      "nonConnected[" +
+      "/* not connected vertices */" +
+      "]" +
+      "halfConnected[" +
+      "/* half connected graph */" +
+      "(v3 {id:3, value:\"D\"})" +
+      "(v0)-[e6]->(v1)" +
+      "(v0)-[e7]->(v2)" +
+      "(v0)-[e8]->(v3)" +
+      "(v1)-[e9]->(v2)" +
+      "]";
+
+    this.loader = getLoaderFromString(graphString);
+  }
+
+  /**
+   * Runs the specified tests
+   */
+  @Test
+  public void runTests() throws Exception {
```

Comment:
This method is not necessary. Please add @Test annotation to all functions below and make them public.

Reply:
Changed it